content/docs/rfcs/19/README.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: 19
+title: 19/WAKU2-DIRECT-MESSAGE
+name: Waku v2 Direct Message
+status: raw
+editor: Franck Royer <franck@status.im>
+contributors:
+---
+
+This specification explains the Waku v2 Direct Message protocol which enables a peer to send a direct message to another peer using the Waku v2 network.
```

Comment:
I'd probably making this into something more Ethereum specific

Reply:
Happy to do so and not fall in the "generalised too early" trap. Interestingly enough, only the derivation is Ethereum (actually ecdsa) specific.

Reply:
Well, as anywhere else "Ethereum" is mentioned and not the actual key scheme used, the same could be should be done here. Or the other way around and remove Ethereum everywhere.

Reply:
true, ECDSA messaging? :D